Gemfile

```diff
@@ -57,6 +57,7 @@ gem 'sitemap', git: 'https://github.com/ualbertalib/rails-sitemap.git'
 gem 'rest-client'
 
 gem 'noid', '~> 0.8'
+gem 'nokogiri', '~> 1.7.1'
```

Comment:
Like this:

https://github.com/ualbertalib/HydraNorth/blob/496dcc631f7efc4b028b6f3515b45207182ab90b/Gemfile#L21-L22